src/page-manager/wrapper.js

```diff
@@ -0,0 +1,173 @@
+/**                   _
```

Comment:
I think we could keep this file in the root `src` folder, and just name it `controller-wrapper.js`. The repo is actually small and once we remove the files that doesn't belong here (like token-input) there will be just like 4-5 files